Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingAccelerationStructurePass.h

```diff
@@ -43,11 +43,20 @@ namespace AZ
             void BuildInternal() override;
             void FrameBeginInternal(FramePrepareParams params) override;
 
+            // Used to set some build options for the TLASes
+            static AZ::RHI::RayTracingAccelerationStructureBuildFlags CreateRayTracingAccelerationStructureBuildFlags(bool isSkinnedMesh);
+
             // buffer view descriptor for the TLAS
             RHI::BufferViewDescriptor m_tlasBufferViewDescriptor;
 
             // revision number of the ray tracing data when the TLAS was built
             uint32_t m_rayTracingRevision = 0;
+
+            // keeps track of the current frame to determine updates or rebuilds of the skinned BLASes
+            uint32_t m_frameCount = 0;
+
+            // the bits of this constant are used to check if a skinned BLAS is going to be rebuilt in any given frame
+            static constexpr uint32_t SKINNED_BLAS_REBUILD_FRAME_INTERVAL = 8;
```

Comment:
This could be made configurable.

Reply:
It might be helpful to make it configurable. But I am okay it's not in this change.

Reply:
I have decided to add this configurability in a later PR. I will resolve this thread, but if there are any objections feel free to reopen this.